src/xpath.js

```diff
@@ -4,14 +4,14 @@ define([
     xpath
 ) {
     return {
-        makeXPathModels: function (hashtagToXPath, hashtagToTransformation) {
+        makeXPathModels: function (form) {
```

Comment:
It's a little weird that this takes a `form` parameter that isn't necessarily a form, but rather an object that has properties for `hashtagDictionary` and `hashtagTransformations`.